database/CoreDatabaseManager.java

```diff
@@ -169,4 +179,30 @@ public void close() {
     protected static boolean isReservedName(String name) {
         return name.startsWith(RESERVED_NAME_PREFIX);
     }
+
+    protected Metrics.DatabaseDiagnostics databaseDiagnostics(
+            TypeDB.Database database, Metrics.DatabaseSchemaLoad schemaLoad, Metrics.DatabaseDataLoad dataLoad
+    ) {
+        return new Metrics.DatabaseDiagnostics(database.name(), schemaLoad, dataLoad, true);
+    }
```

Comment:
Does this really warrant a helper at this point? It could be inlined.

Reply:
It's not necessary, but not a bad thing for symmetry, I guess. At least that's why I left it this way.

Reply:
Symmetry with what? This is only used in `submitDatabaseDiagnostics()`, which doesn't use any similar helpers.
This is indirection for the sake of indirection.

Reply:
Yep, I did think that this method is used in another place. Fixed.

Reply:
Oh! I thought it was a different method.
This indirection is needed for `TypeDB Cloud` to override only the diagnostics creation, not the logic of getting the stats!